src/YesSql.Provider.MySql/MySqlDbProviderOptionsExtensions.cs

```diff
@@ -27,7 +35,7 @@ public static class MySqlDbProviderOptionsExtensions
             {
                 ConnectionFactory = new DbConnectionFactory<MySqlConnection>(connectionString, true),
```

Comment:
There's only one constructor https://github.com/sebastienros/yessql/blob/b6f5e719a1c2147d599c705bcb23057561e32f94/src/YesSql.Core/Services/Configuration.cs#L42 that requiring `reuseConnection`, so how we can remove it?!!